mllib/src/main/scala/org/apache/spark/mllib/tree/model/treeEnsembleModels.scala

```diff
@@ -166,6 +156,56 @@ class GradientBoostedTreesModel(
 
 object GradientBoostedTreesModel extends Loader[GradientBoostedTreesModel] {
 
+  /**
+   * Method to compute initial error and prediction as a RDD for the first
+   * iteration of gradient boosting.
+   * @param data RDD of [[org.apache.spark.mllib.regression.LabeledPoint]]
```

Comment:
Doc: "Training data"  The generated doc will already include the argument type.  Also a leftover problem from before...